packages/sdk-btc/source/transaction.dto.ts

```diff
@@ -17,7 +17,7 @@ export class ConfirmedTransactionData extends DTO.AbstractConfirmedTransactionDa
 	}
 
 	public override confirmations(): BigNumber {
-		return BigNumber.ZERO;
+		return this.bigNumberService.make(this.data.confirmations);
```

Comment:
```suggestion
		return BigNumber.make(this.data.confirmations);
```

Confirmations don't have the need for decimal handling.